terraform/Azure/only_private_blob_storage/only_private_blob_storage.rego

```diff
@@ -0,0 +1,19 @@
+package torque
+
+import input as tfplan
+
+# --- Validate blob storage are private ---
+
+deny[reason] {
+    allowed_set:= { "private" }
+    results_set:= { r | r:= tfplan.resource_changes[_].change.after.container_access_type }
+    diff:= results_set - allowed_set
+    
+    print("allowed_set:       ", allowed_set)
```

Comment:
Please consider setting in comments/remove.. 
I don't know how it will be reflected in/will affect the TF runner, and anyway, it's not related to the main logic